docs/en/setup/backend/backend-cluster.md

```diff
@@ -82,6 +83,14 @@ The following setting are provided to set the host and port manually, based on y
 - internalComHost, the host registered and other oap node use this to communicate with current node.
 - internalComPort, the port registered and other oap node use this to communicate with current node.
 
+## Nacos
+Set the **cluster/selector** to **nacos** in the yml to enable.
+
+```yaml
+cluster:
+  selector: ${SW_CLUSTER:nacos}
+  # other configurations
```

Comment:
Please list the configuration of Nacos, and explain them. Also, Nacos should be at the last one.